web/styles/zulip.css

```diff
@@ -737,6 +738,7 @@ body {
     --color-exit-button-border: hsl(0deg 0% 0% / 60%);
     --color-exit-button-background: hsl(211deg 29% 14%);
     --color-exit-button-background-interactive: hsl(211deg 29% 17%);
+    --color-border-new-style-button-focus: hsl(0deg 0% 70%);
```

Comment:
Neither of the colors set here matches what we had before -- so this is a design change -- is that intended? What other buttons does that change apply to?

I'm not sure how to review this.

Reply:
@timabbott you're right, this affected many buttons like `preview profile`, `add a new bot` etc. Now I've updated the code so that it only affects the `Subscribe` button.